metadata-ingestion/setup.py

```diff
@@ -152,6 +152,27 @@ def get_long_description():
     },
     entry_points={
         "console_scripts": ["datahub = datahub.entrypoints:datahub"],
+        "datahub.ingestion.source": [
+            "file = datahub.ingestion.source.mce_file",
```

Comment:
Two minor nits on this:
1. I'd prefer to reference the class directly using the attrs param - that way, you can define multiple sources with different names in the same module
2. Some of these are required to load, whereas others are ok if the load fails. Given that setuptools already defines the `load()` method, I don't want to change the syntax. Perhaps we can just have some asserts in source_registry.py and sink_registry.py to ensure that the required sources/sinks are loaded